docs/search.md

```diff
@@ -44,141 +44,605 @@ The keyword combined with spatial and temporal parameters are used to search for
+                        "shape": {
+                            "type": "polygon",
+                            "coordinates": coords
+                         },
+                         "relation": "within"
+                    }
+               }
+          }
+     }
+}
 ```
 The filter has been nested within a `boolean/must match` query so that it completely matches the documents being queried. The filter has been applied to the `metadata.o2r.spatial.geometry` field of the dataset with a `within` relation so that the ERC that matches the document's coordinates completely are fetched.
 
 #### Parameters for Keyword Searching 
 `keyword` :- Any word, phrase in the form of text that should be used to match the title of ERC.
 A simple match query is used at the back-end to match the query value with the values in dataset
-```bash
+```json
 {
     "match" : { "metadata.o2r.title" : keyword}
   }
   ```
-  ##### NOTE :
-  The parameters should be used together and not seperatley as they have been coupled together to form one single boolean query as stated below so that a refined search is possible and ERC which passes all the criteria are fetched.
-  
-```bash
-{
-                "query": {
-                    "bool": {
-                        "must": [{
-                                "range": {
-                                    "metadata.o2r.temporal.begin": {
-                                        "from": from
-                                    }
-                                }
-                            },
-                            {
-                                "range": {
-                                    "metadata.o2r.temporal.end": {
-                                        "to": to
-                                    }
-                                }
-                            },
-                            {
-                                "bool": {
-                                    "must": {
-                                        "match_all": {}
-                                    },
-                                    "filter": {
-                                        "geo_shape": {
-                                            "metadata.o2r.spatial.geometry": {
-                                                "shape": {
-                                                    "type": "polygon",
-                                                    "coordinates": coords
-
-
-                                                },
-                                                "relation": "within"
-                                            }
-                                        }
-                                    }
-                                }
-                            },
-                            {
-    "match" : { "metadata.o2r.title" : keyword }
-  }
-                        ]
-                    }
-                }
-            }
-```
+ 
 ## Suggesters - WORK IN PROGRESS
```

Comment:
Please remove the section on Suggesters, we don't plan this feature at the moment.